unittest/mid_hook.cpp

```diff
@@ -14,7 +14,7 @@ TEST_CASE("Mid hook to change a register", "[mid_hook]") {
 
     struct Hook {
         static void add_42(SafetyHookContext& ctx) {
-#if defined(_M_X64)
+#ifdef _M_X64
             ctx.rcx = 1337 - 42;
 #elif defined(_M_IX86) 
```

Comment:
I prefer:

```c++
#if defined(_M_X64)
...
#elif defined(_M_IX86)
...
#endif
```

Over:

```c++
#ifdef _M_X64
...
#elif defined(_M_IX86)
...
#endif
```

Because it's more consistent.